internal/ethapi/api.go

```diff
@@ -1238,10 +1238,11 @@ func DoEstimateGas(ctx context.Context, b Backend, args TransactionArgs, blockNr
 		balance := state.GetBalance(*args.From) // from can't be nil
 		available := new(big.Int).Set(balance)
 		if args.Value != nil {
-			if args.Value.ToInt().Cmp(available) >= 0 {
-				return 0, core.ErrInsufficientFundsForTransfer
+			value := args.Value.ToInt()
+			if value.Cmp(available) >= 0 {
+				return 0, fmt.Errorf("%w: address: %s, have: %v, want: %v+(gas)", core.ErrInsufficientFunds, *args.From, available, value)
```

Comment:
I don't know why it says `(gas)` ? The unit is wei 
```suggestion
				return 0, fmt.Errorf("%w: address: %v, have: %v, want: >%v", core.ErrInsufficientFundsForTransfer, *args.From, available, value))
```

Reply:
Sorry, I intended to say that there was an extra gasFee here. maybe changed into something like below:


```diff
-                               return 0, fmt.Errorf("%w: address: %s, have: %v, want: %v+(gas)", core.ErrInsufficientFunds, *args.From, available, value)
+                               return 0, fmt.Errorf("%w: address: %s, have: %v, want: %v+(gasFee)", core.ErrInsufficientFunds, *args.From, available, value)

```